core/xml.ts

```diff
@@ -478,6 +491,34 @@ export function domToWorkspace(xml: Element, workspace: Workspace): string[] {
   return newBlockIds;
 }
 
+/** Deserializes the given comment state into the given workspace. */
+function loadWorkspaceComment(
```

Comment:
I am curious why not have this implemented as a function on the class itself, like it used to be? It seems like that would make it easier on people who are implementing custom subclasses, and it would make more sense organizationally IMO. And we do have a pattern of a `fromXml` method on fields, events, etc.

Reply:
No one is allowed to implement custom subclasses atm. We discussed that and decided YAGNI.

Reply:
Fair enough :)